pt/The-Manual/Tutorials-Examples/Simple-Acl-controlled-Application.rst

```diff
@@ -30,7 +30,7 @@ Preparando sua Aplicação
 Primeiro, vamos pegar uma cópia recente do Cake.
 
 Para obter um download recente, visite o projeto CakePHP no Cakeforge:
-http://cakeforge.org/projects/cakephp/ e faça o download da versão
+https://cakeforge.org/projects/cakephp/ e faça o download da versão
```

Comment:
Doesn't look like cakeforce.org supports https.  Also, these links seem to be invalid and open an online poker site now.

Should we remove them? @markstory

Reply:
Yeah cakeforge has been dead for at least 8 years.

Reply:
remove